SG14/transcode.h

```diff
@@ -89,7 +89,7 @@ sg14::rsa_encryptor<OutIt>::rsa_encryptor(OutIt&& dest_, int m_, int n_)
 template<typename OutIt>
 sg14::rsa_encryptor<OutIt>& sg14::rsa_encryptor<OutIt>::operator=(const char& element)
 {
-	OutIt::operator=(element + m);
+	OutIt::operator=(static_cast<const char>(element + m));
```

Comment:
Can it be just `char` instead of `const char`?

Reply:
sure